src/components/App/App.tsx

```diff
@@ -1,26 +1,42 @@
-import { Switch, Route } from 'react-router-dom';
+import { useEffect, useState } from 'react'
+import { Switch, Route, Redirect } from 'react-router-dom';
 import './App.css';
 import Splash from '../Splash/Splash'
 import Dashboard from '../Dashboard/Dashboard'
 import GameScreen from '../GameScreen/GameScreen'
 
 
 const App = () => {
+
+  const [userName, setUserName] = useState<string>('')
+  const [userKey, setUserKey] = useState<string>('')
+
+useEffect(() => {
+  const activeUser = localStorage.getItem('chessAdventureName') || ''
+  const activeKey = localStorage.getItem('chessAdventureKey') || ''
+  setUserName(activeUser)
+  setUserKey(activeKey)
+}, [])
+
```

Comment:
This is the rehydrating of state that happens from local storage when a user returns to the site while logged in. We pull information from local storage and this will determine any redirecting that happens.

Reply:
And there is no logout at the moment, right? So if a user makes an account, that is what they are stuck with barring clearing local storage.